p2p/base/port.cc

```diff
@@ -883,7 +884,7 @@ void Port::OnNetworkTypeChanged(const rtc::Network* network) {
 
 std::string Port::ToString() const {
   rtc::StringBuilder ss;
-  ss << "Port[" << rtc::ToHex(reinterpret_cast<uintptr_t>(this)) << ":"
+  ss << "Port[" << record_replay_id_ << ":"
```

Comment:
Hmm, this is kind of a substantial change and will also change behavior in the non-recording case so that the strings are no longer unique (since NewIdAnyThread returns zero if we're not recording) which could break things unpredictably.

Instead of this (and the related changes in the PR), I think it would be better to record/replay the value of `reinterpret_cast<uintptr_t>(this)` which should be alright unless this is invoked non-deterministically.